src/ray/protobuf/common.proto

```diff
@@ -268,6 +271,7 @@ message ActorDeathCause {
 // Indicates that this actor is marked as DEAD due to runtime environment setup failure.
 message RuntimeEnvFailedContext {
   // TODO(sang,lixin) Get this error message from agent.
+  // This error_message is deprecated. Use the RayErrorInfo.error_message() instead.
```

Comment:
I still see a lot of error_messages from actors. How difficult is it to take it out in this PR?

Reply:
Seems we still need it since `ActorDeathContext` has been used independently in parts of the code, for example, it's part of the actor table data.

Reply:
Hmm in this case, do you think we should remove the outermost error_message field? (to be consistent). Don't need to be done in this PR, but I wonder if that's cleaner

Reply:
that would mean we have to do all the if...else context when we consume this, which is the original motivation to have the outer `error_message` field. 

I would actually propose removing the `error_message` from that actor-related context, but maybe add a helper method that could generate the error message on the fly given the context.

Reply:
I see. I don't have special preference as long as it is consistent. Let's probably handle in a follow up (but not a high priority)